lib/pushmi_pullyu/swift_depositer.rb

```diff
@@ -23,27 +23,30 @@ def deposit_file(file_name, swift_container)
 
     era_container = swift_connection.container(swift_container)
 
-    deposited_file = if era_container.object_exists?(file_base_name)
-                       era_container.object(file_base_name)
-                     else
-                       era_container.create_object(file_base_name)
-                     end
+    object_exists = era_container.object_exists?(file_base_name)
```

Comment:
Assigned a variable but never used more then once. Do you need this? Does it improve code readability? Probably not needed then?